docs/src/content/pages/reader-api.mdoc

```diff
@@ -52,13 +52,41 @@ const post = await reader.collections.post.read(slug);
 You can get an array of objects containing **both** slug and entry data for a collection with:
 
 ```javascript
-const entry = await reader.collections.{collectionName}.all();
+const entries = await reader.collections.{collectionName}.all();
 
 // Example
-const blog = await reader.collections.blog.all();
+const posts = await reader.collections.blog.all();
 ```
 
-### For singletons
+### Filtering based on a field's value
+
+A common use case is to filter the entries based on a field's value before rendering them in the UI. You can use Keystatic's reader functions along with standard JS array methods like `.filter()` and `.sort()` to achieve this pattern.
+
+Let's say you're building a list page for blog posts, and the collection schema for each entry defines a `draft` checkbox field:
```

Comment:
Just to keep the scenario super obvious, and highlight _why_ you would want to filter the posts, it would be good to also say that the list page only shows published posts. Something like:

```md
Let's say you're building a list page for blog posts, and you only want to display published posts. The collection schema for each entry defines a `draft` checkbox field, indicating a not-to-be-published post:
```

The schema entry could tie in the scenario with repeated language (i.e. this would be line 73):

```js
draft: fields.checkbox({
  label: 'Convert to draft,
  description: 'Prevent publishing of this post. This can be changed later.',
  defaultValue: false,
})
```

Let me know what you think.

Reply:
Great suggestions!! Thanks Laney, will update